### PR TITLE
Outdated velocity

### PR DESCRIPTION
**Pull request by oversword**
After getting to test this properly on 5.3 it seems the `get_player_velocity` method only returns something useful on 5.4
This MR feature tests for `get_velocity` and also includes fallback velocity values
This would previously crash on 5.3 when you dropped an item (oops...)
This has been tested on 5.3 and definitely works now

### CODE REVIEW

**Comment by oversword** [comment]
@SmallJoker My bad, definitely actually working now :D

File: init.lua [hunk 1]
```diff
@@ -110,7 +118,7 @@ local function entity_pos(obj, offset)
 					obj:get_pos()
 				),
 				vector.multiply(
-					obj:get_player_velocity(),
+					velocity,
```
Comment (SmallJoker) [inline init.lua:121]: `velocity` must be a vector (table).  When `get_*velocity` returns `nil`, it means that the object is no longer valid (i.e. deleted).

Comment (oversword) [reply init.lua:121]: Oh you're right, I don't know how I didn't notice that, it hasn't used the fallback values yet though so the condition is working, I'll fix now

Comment (oversword) [reply init.lua:121]: @SmallJoker  that went past me, fluxionary and luk3yx and none of us caught it :facepalm: should be good now

Comment (oversword) [reply init.lua:121]: (( honestly I hate this entire section of code, it should just be `math.round(obj:pos())`, I don't know why we need to project velocity, but I suppose on a server it's better than lagging behind ))

Comment (SmallJoker) [reply init.lua:121]: To simplify things, you could let `velocity` be `nil` on error and change this line to
`velocity or { x=0, y=0, z=0 }` which makes the fallbacks above superfluous.

Comment (oversword) [reply init.lua]: @SmallJoker That's changed now, much cleaner, thanks

**Review by SmallJoker — APPROVED** [review]
LGTM. Will merge in a few days.

**Pull request merged**